### DISCUSSION TITLE
22000.493.42.11 Triggers Webroot

### DISCUSSION THREAD

**Discussion by DustyMustard** [valinet/ExplorerPatcher#867] [Q&A]
Heads up that something changed in your release right before 22000.493.42.11 to that version that Webroot trips out and I have to whitelist.

![2022-02-15 18_30_10-Webroot SecureAnywhere](https://user-images.githubusercontent.com/710289/154185814-4deafe92-bc71-40b3-bdab-a274032620fb.png)

**Comment by Poopooracoocoo** [comment]
You've gotta report it as a false positive to Webroot.

**Reply by DustyMustard** [reply]
Aye, submitted them [here](http://snup.webrootcloudav.com/SkyStoreFileUploader/upload.aspx#) for the following.

Update for ExplorerPatcher from https꞉∕∕github.com∕valinet∕ExplorerPatcher∕releases∕download∕22000.493.42.11_61626d9∕ep_setup.exe
Update for ExplorerPatcher from https꞉∕∕github.com∕valinet∕ExplorerPatcher∕releases∕download∕22000.493.42.12_2efe2a6∕ep_setup.exe
Update for ExplorerPatcher from https꞉∕∕github.com∕valinet∕ExplorerPatcher∕releases∕latest∕download∕ep_setup.exe

**Comment by valinet** [comment]
Some things definitely changed, the commit history on the Git repo also tells you that.

**Reply by DustyMustard** [reply]
Yes, still want to let you know.

**Reply by valinet** [reply]
Thanks, but there's nothing I can do on my side regarding bugs in antivirus products (or "false positives" as the vendors call them) other than to digitally sign the program, which would probably please all of them.

**Reply by DustyMustard** [reply]
Maybe take up donations to pay for digitally signing? I'm sure I am not the only one that would chip in a few shekels.

**Reply by valinet** [reply]
Yeah, I take donations, there's a link on the main page, a couple of folks donated but it feels like... idk, if you do a job and someone wants to say "thanks" to you and gives you say $1, how do you feel the Government taking a cut because they just sit and simply exist (by definition Romanians don't trust the Gov't, so no amount of reasoning can convince one otherwise), then the landlord because without a roof on top of your head you wouldn't have been able to work, then the mafia because why not, and so on until you're left with a couple of cents. You kind of question why bother, you know what I mean, it brings you down in a way, idk how to say...

Some things you can't do anything about them, you pay up and that's it (the Gov't, via the usual taxes when cashing out stuff etc), some of them suck but you put up with them as well because the alternative would be cumbersome (the landlord, in this case PayPal, which actually takes a hefty commission for the amount of nothing they do - a landlord at least provides you with an actual something, i.e. a home to live in - not the best comparison maybe but let's roll with it), and then there's the mafia, which shouldn't be a thing but that's what happens when things are rotten. In this case, the antivirus companies, and even Windows itself are the mafia, and a digital certificate is just a protection tax for them.

Does a digital certificate actually enhance security in this case? Absolutely not. The files are built on clean servers in the cloud, directly based off the source code available online. If that's compromised, even with a certificate, it would still be compromised, as something malicious would then get signed. So no, it's just "protection tax" basically in order to please this conglomerate of giants that really enjoy stifling homebrew development. That's exactly why they do it. That's the thing, it's yet another method to kind of control what people run on their computers. Even this "false positive" term is shit. If I have a problem in EP, it's called a bug. If an antivirus engine thinks EP is a virus, which clearly is not, then it's a "false positive". Every other software calls their bugs what they are - bugs (i.e. things that don't work right), only antivirus engines call their bugs "false positives".

So I feel like putting money towards buying a certificate is wasted money. And I feel kind of itchy about wasting money, especially when it comes to rightfully earned and hard worked on money. That's the main deterrent now. Idk, I really feel like generally is just a waste of money, feel like giving in to this bullying...
